lib/AcqList/utils/queryUtils.js

```diff
@@ -0,0 +1,53 @@
+import { omit } from 'lodash';
+
+import {
+  SEARCH_PARAMETER,
+  ASC_DIRECTION,
+  SORTING_PARAMETER,
+  SORTING_DIRECTION_PARAMETER,
+} from '../constants';
+
+export const buildFilterQuery = (queryParams, getSearchQuery) => {
+  const filterParams = omit(queryParams, [SORTING_PARAMETER, SORTING_DIRECTION_PARAMETER]);
+
+  return Object.keys(filterParams).map((filterKey) => {
+    const filterValue = queryParams[filterKey];
+
+    if (!filterValue) return false;
+
+    if (filterKey === SEARCH_PARAMETER && filterValue) {
```

Comment:
Do you really need this `&& filterValue` condition ?

Reply:
yes, to not populate empty value

Reply:
But in line `16` you check `filterValue` on empty value... and in this case you get non empty value every time. or I'm wrong?